### PR TITLE
fix : reset button is disabled by default

### PR DESCRIPTION
**Pull request by felixbbertrand**

### CODE REVIEW

File: packages/react-app/src/components/filter.tsx [hunk 1]
```diff
@@ -61,6 +64,11 @@ export function Filter({ compact }: Props) {
   const { below } = useViewport();
   const states = [ENUM_QUEST_STATE.All, ENUM_QUEST_STATE.Active, ENUM_QUEST_STATE.Expired];
   const { isFilterShown } = useFilterContext();
+  const isFilteringOriginalState =
```
Comment (Corantin) [inline packages/react-app/src/components/filter.tsx:67]: I'd make a useEffect and compare actual filter value with the default one in constant constants.ts->DEFAULT_FILTER

**Review by Corantin — CHANGES_REQUESTED** [review]
Bien Vu :p better like that

File: packages/react-app/src/components/filter.tsx (hunk 1 above)
Comment (Corantin) [reply packages/react-app/src/components/filter.tsx:67]: So if we change the default config, no need to adapt this code

Comment (felixbbertrand) [reply packages/react-app/src/components/filter.tsx]: true, good idea

**Review by Corantin — APPROVED** [review]

**Pull request merged**